Sources/SWBCore/DependencyResolution.swift

```diff
@@ -206,7 +206,17 @@ struct SpecializationParameters: Hashable, CustomStringConvertible {
     func imposed(on parameters: BuildParameters, workspaceContext: WorkspaceContext) -> BuildParameters {
         var overrides = parameters.overrides
         if let sdkRoot {
-            overrides["SDKROOT"] = sdkRoot
+            // The sdkCanonicalName from a Platform is not guaranteed to be available
+            let defaultSDKAvailable: Bool
+            do {
+                let foundSDK = try workspaceContext.core.sdkRegistry.lookup(sdkRoot, activeRunDestination: parameters.activeRunDestination)
+                defaultSDKAvailable = foundSDK != nil
+            } catch {
+                defaultSDKAvailable = false
+            }
+            if defaultSDKAvailable {
+                overrides["SDKROOT"] = sdkRoot
+            }
```

Comment:
I'm not 100% sure about this. The overridden `SDKROOT` was always `platform.sdkCanonicalName`, which is always the same as `platform.name` (`"webassembly"`), but `"webassembly"` SDK is unavailable.